packages/flutter/test/material/tabs_test.dart

```diff
@@ -386,6 +386,37 @@ void main() {
       material3 ? const Size(14.25, 72.0) : const Size(14.0, 72.0));
   });
 
+  testWidgets('Tab icon-label margin', (WidgetTester tester) async {
+    Widget buildApp({required bool useMaterial3}) {
+      final ThemeData theme = ThemeData(fontFamily: 'FlutterTest', useMaterial3: useMaterial3);
+      return MaterialApp(theme: theme, home: const Center(child: Material(
+        child: Tab(
+          icon: Icon(Icons.house),
+          text: 'x',
+        )
+      )));
+    }
```

Comment:
This is breaking multiple format rules.

Reply:
I was trying to separate the important piece, which is the Tab. I went with a separate variable instead in the last commit